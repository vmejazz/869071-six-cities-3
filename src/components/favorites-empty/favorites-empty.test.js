import React from "react";
import renderer from "react-test-renderer";
import FavoritesEmpty from "./favorites-empty.jsx";

it(`<FavoritesEmpty /> should render favorites empty page `, () => {
  const tree = renderer
      .create(
          <FavoritesEmpty
          />
      )
      .toJSON();

  expect(tree).toMatchSnapshot();
});
