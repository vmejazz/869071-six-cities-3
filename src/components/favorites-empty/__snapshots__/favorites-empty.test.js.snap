// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<FavoritesEmpty /> should render favorites empty page  1`] = `
<section
  className="favorites favorites--empty"
>
  <h1
    className="visually-hidden"
  >
    Favorites (empty)
  </h1>
  <div
    className="favorites__status-wrapper"
  >
    <b
      className="favorites__status"
    >
      Nothing yet saved.
    </b>
    <p
      className="favorites__status-description"
    >
      Save properties to narrow down search or plan yor future trips.
    </p>
  </div>
</section>
`;
